components/Nav/NavMobile/NavMobile.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import Link from 'next/link';
+import { donateLink, s3 } from 'common/constants/urls';
+import { navItems } from 'common/constants/navigation';
+import flattenDepth from 'lodash/flattenDepth';
+import HamburgerIcon from 'static/images/icons/hamburger.svg';
+import styles from './NavMobile.css';
+
+export default class NavMobile extends Component {
+  constructor(props) {
+    super(props);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.handleMouseLeave = this.handleMouseLeave.bind(this);
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+  }
+
+  state = {
+    dropdownShow: false,
+  };
+
+  // flattens navItems
+  links = () => flattenDepth(navItems.map(navItem => [navItem, navItem.sublinks]), 2);
```

Comment:
I see what you're trying to do here. Instead of calling `links()` in the JSX, just do this at the top of the render method:

```javascript
const linksList = flattenDepth(navItems.map(navItem => [navItem, navItem.sublinks]), 2);
```

and in your JSX, just map off of `linksList.map()`

Reply:
Aight, cool